### ISSUE TITLE
[BUG] micro-tethys-platform requires django4.2

### ISSUE THREAD

**Issue by sdc50** [tethysplatform/tethys#1108]
When installing the latest dev build of `micro-tethys-platform` it will only install with `django=4.2` (i.e. not `django=4.2.*`). 

```
Looking for: ['micro-tethys-platform=4.2.0.post3.dev62', 'django=4.2.16']

tethysplatform/label/dev/osx-arm64                          Using cache
tethysplatform/label/dev/noarch                             Using cache
conda-forge/osx-arm64                                       Using cache
conda-forge/noarch                                          Using cache
Encountered problems while solving:
  - package micro-tethys-platform-4.2.0.post3.dev62+gda1d12a9.d20241028-dev62_gda1d12a9 requires django 4.2, but none of the providers can be installed
```

When I unpin `micro-tethys-platform` it resolves to `micro-tethys-platform-4.2.0.post3.dev26`, so something happened since then that prevents the patch versions of `django` to be compatible.

**Comment by sdc50** [comment]
I think the `==` in the `install_tethys.sh` is causing the issue. 

```bash
        echo "Updating environment.yml Django version ${DJANGO_VERSION}..."
        sudo sed -i.bak "s/django>=.*/django==${DJANGO_VERSION}/" "${TETHYS_SRC}/environment.yml"
        sudo sed -i.bak "s/django>=.*/django==${DJANGO_VERSION}/" "${TETHYS_SRC}/micro_environment.yml"
```

If we replace it with just `=` then it will allow other patch versions of `django=4.2`. However, I think we want the conda packages to be installed with several versions of `django`/`version`. Since the conda recipe is derived from the `environment.yml` file I think we need to not modify the file, but just update the environment after creating it.

**Comment by sdc50** [comment]
fixed by #1109
